guides/release/state-management/computed-properties.md

```diff
@@ -1 +1,634 @@
-Coming Soon!
+Computed properties are a concept in Ember prior to Octane. They serve the same
+purpose as tracked properties and native getters, allowing users to respond to
+changes, derive state, and ultimately update the DOM. They also have built-in
```

Comment:
Pulling in a prior definition from [here](https://guides.emberjs.com/release/object-model/computed-properties/#toc_computed-properties-only-recompute-when-they-are-consumed)

```suggestion
changes, derive state, and ultimately update the DOM. 
In a nutshell, computed properties let you declare functions as properties.
You create one by defining a computed property as a function, which Ember will automatically call when you ask for the property. 
They also have built-in
```